enterprise/websvc.rest/test/unit/src/org/netbeans/modules/websvc/rest/codegen/license-default.txt

```diff
@@ -1,3 +1,22 @@
+/*
```

Comment:
In your first version of the PR, this was a freemarker comment. Given, that this file is explicitly referenced as a template, it would be good to switch back to it. I doubt, that it has serious impact (it is in the unittest sources), but new coders could look at it and copy from it. Please restore to your first version.